### DISCUSSION TITLE
Workshop Ideas

### DISCUSSION THREAD

**Discussion by BalduinLandolt** [network-digital-old-norse-studies/network-digital-old-norse-studies.github.io#10] [Ideas]
Here we can share and discuss ideas for future network workshops.

I'm not sure about this format - maybe it's nice to have it all in one thread, maybe we should have a thread per idea... we'll have to see, let's try it out.

**Comment by BalduinLandolt** [comment]
During the SQL Introduction by @tarrinw we already came up with some ideas:

- regex
  - who would want to do that?
- Graph Databases
  - Neo4J (@kraus-s ?)
  - RDF (@BalduinLandolt ?)
- XPath
  - who would want to do that?
- SQL Part II
  - @tarrinw again?
  - as proposed, this should be project based: we would have to know what data we want to work with and what we want to get out of the data

**Reply by SvenjaWa** [reply]
If there's an interest in corpus tools, I've previously annotated historical language data in Exmaralda. It has its downsides but it's  an easily accessible way to create xml-based data

**Reply by kraus-s** [reply]
I'd be keen to give an intro to graph DBs :)

**Comment by BalduinLandolt** [comment]
At some point I also offered to give a short introduction to Git and GitHub. Neither is really as complicated as most tutorials make it sound, but I've seen people being hesitant to get started with it, because it comes with a lot of new concepts all at once.

So far it seemed that people weren't overly interested in this workshop topic... but of course the offer still stands!

**Reply by SvenjaWa** [reply]
I'm still interested! (:

**Comment by BalduinLandolt** [comment]
Another topic we should at some point cover: IIIF  
Any ideas who might be best to give such a talk? One of us? Or someone external?

**Reply by BalduinLandolt** [reply]
Some options who could give such a presentation:
- @SvenjaWa has contacts to Corvey Digital
- Maybe Bill, as the Fragmentarium makes heavy use of IIIF?
- At the DaSCH we have a PhD candidate who's working with IIIF a lot - I could ask him

**Comment by BalduinLandolt** [comment]
In terms of database technologies, we already covered _SQL_ and we are planning _Graph Databases_; the logical conclusion would be to also cover _NoSQL Databases_ and _Event Sourcing_ as the two remaining important database management system categories.

**Comment by kraus-s** [comment]
Dr. Anna Dorofeeva in Göttingen might be interesting for us, she has a strong DH focus and works on medieval latin codicology and paleography. I think we should invite her for a talk.

**Comment by ElineElmiger** [comment]
I'd be interested in hearing about and discussing non-linear / non-traditional publishing formats. What alternatives are there to PDFs, which is basically just digital paper? If we put data online, how can we make sure it stays accessible and citable? Can we make tools where users can play around with our data? How can we present research in a non-linear way while it stays understandable (digital "tour" through data)?

I know Sven has some experience with/plans on publishing DH-data online, don't know about anyone else.

**Reply by BalduinLandolt** [reply]
That sounds like a great topic!  
I would be happy to contribute a 10 minute section on what our take at the DaSCH is, with regards to publishing research data. But the overall workshop should be done by someone who knows more about this topic than I do. :)

**Comment by MaditaVerena** [comment]
In our Network meeting today, a few websites were mentioned that - as promised - I've collected:

- [Resonating networks](https://www.resonatingnetworks.com/) (a project starting soon at the Universität Basel and Humboldt-Universität zu Berlin; among others Balduin, Eline and Tim are involved)
- [Collection of Swiss Law Sources online](https://www.ssrq-sds-fds.ch/en/home/) (a project mentioned by Julianne)
- [catma](https://catma.de/) (online annotation software mentioned by Nora and Balduin)
- [CfP for the next Digital Humanities in the Nordic and Baltic Countries conference](https://dhnb.eu/conferences/dhnb2024/cfp/) (DHNB 2024 in Reykjavík; deadline for submissions is 21 January 2024)

**Reply by BalduinLandolt** [reply]
Two more things that were mentioned during the meeting: 1)  Guðrún Laufey has offered her expertise in hymns (maybe this could be combined with a discussion on digital music encoding, a topic which we haven't really talked about in this group since the Anton Webern presentation in Basel). And 2) Tarrin has mentioned the possibility of PhD level courses in Copenhagen as a vessel for certain activities.  
(I'm sure there was more, but I don't remember right now.)
